### PR TITLE
Postprocessing videos

### PR DESCRIPTION
**Pull request by leclem**
Allowing user to pass a function testing the output video and discarding it in case it doesn't fit to these rules

### CODE REVIEW

File: .env.config.dev [hunk 1]
```diff
@@ -15,3 +15,4 @@ NB_WORDS_PER_SUBTITLE = 15
 NB_SECONDS_PER_WORDS = 0.5
 DEFAULT_BACKGROUND_MUSIC = medias/royalteefree_background_music.mp3
 MEDIA_POLLING_INTERVAL = 5
+#VIKIT_BACKEND_URI=https://ede8c565-0603-45fb-8899-c27d44dc9dfe-00-6v5f2miz9dgw.janeway.replit.dev/models
```
Comment (jeffmac-aix) [inline .env.config.dev:18]: good catch, this can be deleted if nobody use it

File: script_example.py [hunk 2]
```diff
@@ -372,6 +373,79 @@ async def colabCode():
...
+
+If theere is no human, output False.
+
+Output ONLY True or False nothing else. No other text or characters are allowed. Do not get wrong. Do not output any text.
+
+    """,  video=media_url)
+    is_outside = await ml_models_gateway.ask_gemini(prompt_is_outside, gemini_version)
+
+    if ("True" in is_outside): 
+        print("Outside")
```
Comment (jeffmac-aix) [inline script_example.py:391]: maybe this will be interesting to use this info for debugging in prod, why not using logger instead of print?

Comment (leclem) [reply script_example.py:391]: Yes i forgot to remove the prints from my code, done

**Comment by jeffmac-aix** [comment]
broken down in smaller MR's, closing this one

**Pull request closed without merging**